app/components/tab_bar/tab_bar.html.erb

```diff
@@ -0,0 +1,17 @@
+<nav class="TabBar">
+  <ul>
+    <% items.each do |url, options| %>
+      <li class="TabBar-item <%= 'TabBar-item--active' if options.dig(:active) %>">
+        <a href="<%= url %>">
+          <%= options.dig(:label) %>
```

Comment:
Why `dig` :shrug: and a couple of lines below you use `[:count]`?

Reply:
The `options[:count]` is inside an `if options.key?(:count)` so that key is guaranteed to be present in that case, but I can also replace that with `dig` for consistency…